assets/src/domain/eventEditor/hooks/filterBar/useFilterBarSubscription.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+
+import { useFilterBarService } from '@appLayout/entityList/filterBar';
+
+const useFilterBarSubscription = (): void => {
+	const { subscribe } = useFilterBarService('eventEditor');
+
+	useEffect(() => {
+		const unsubscribe = subscribe(() => {
+			// do some cool stuff here
+		});
+
+		return (): void => {
+			unsubscribe();
+		};
+	}, []);
+};
```

Comment:
This needs a bit of discussion whether we want to keep it in domain or move to application.

Reply:
if it is specific to a particular domain then it would stay here.
if it could be utilized in a totally different application (for example, the bosses come up with a great new idea for some other kind of plugin, or SaaS service, or whatever, and this logic could also be used for that) then it should go into the application folder.

ie:

domain === pertains to Event Espresso (tickets, events, registrations, etc)
application === could be any kind of application (eCommerce, B2B, CRM, analytics, ???)